boards/arm/control_freak/control_freak.dts

```diff
@@ -0,0 +1,296 @@
+/dts-v1/;
+#include <st/f4/stm32f446Xe.dtsi>
+#include <st/f4/stm32f446r(c-e)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+    model = "Control Freak";
+    compatible = "st,stm32f446re-nucleo";
+
+    chosen {
+        zephyr,console = &usart1;
+        zephyr,shell-uart = &usart1;
+        zephyr,sram = &sram0;
+        zephyr,flash = &flash0;
+    };
+    aliases {
+        storage = &w25q128;
+        logfs = &lfs1;
+        debug-serial = &uart4;
+    };
+
+
+    leds: leds {
+        compatible = "gpio-leds";
+        led1: led1{
+            gpios = <&gpiob 0 GPIO_ACTIVE_HIGH>;
+            label = "D1 on board";
+        };
+        led2: led2{
+            gpios = <&gpiob 1 GPIO_ACTIVE_HIGH>;
+            label = "D2 on board";
+        };
+        buzzer: buzzer{
+            gpios = <&gpioa 2 (GPIO_ACTIVE_HIGH )>;
+            label = "buzzer pin";
+        };
+        servo_enable: servo-enable{
+            gpios = <&gpiob 5 GPIO_ACTIVE_HIGH>;
+            label = "Enable pin for the servo";
+        };
+        pump_enable: pump-enable{
+            gpios = <&gpioc 0 GPIO_ACTIVE_HIGH>;
+            label = "Enable pin for pumps";
+        };
+        ldo5v_enable: ldo5v-enable{
+            gpios = <&gpioh 1 GPIO_ACTIVE_HIGH>;
+            label = "Enable pin for 5v ldo";
+        };
+        cam1_enable: cam1-enable{
+            gpios = <&gpioc 14 GPIO_ACTIVE_HIGH>;
+            label = "Enable pin for Camera 1";
+        };
+        cam2_enable: cam2-enable{
+            gpios = <&gpioc 15 GPIO_ACTIVE_HIGH>;
+            label = "Enable pin for Camera 2";
+        };
```

Comment:
Maybe group separately for readability

Reply:
Also should they default active high?

Reply:
I think so? active high means a logical value of 1 brings the voltage up and i believe that is the case for these guys

Reply:
should probably be `GPIO_ACTIVE_HIGH | GPIO_OUTPUT_LOW` though to be explicit about the direction and initial state